torch/_inductor/config.py

```diff
@@ -556,11 +556,14 @@ def use_autoheuristic(name: str) -> bool:
 # Enable indirect_indexing asserts for decompositions and lowerings
 debug_index_asserts = False
 
-# Mode to emulate pytorch eager numerics for lower precision (fp16, bf16)
-# Pytorch eager computes bf16/fp16 by upcasting inputs to fp32 and downcasting after
-# For multiple, fused pointwise nodes, inductor will elide the intermediary upcasts and downcasts
-# Typically this should be closer to fp64 ref numerics. However, it can be useful for debugging
-# to emulate the eager numerics.
+# Mode to emulate PyTorch eager numerics when doing lower precision compute
+# (fp16, bf16).  PyTorch eager computes bf16/fp16 by upcasting inputs to fp32
+# and downcasting after.  When two low precision operators are fused together,
+# Inductor will elide the downcast-upcast pairs (effectively a precision
+# truncation) that would occur between these two operators.  Typically,
+# Inductor's behavior should be closer to fp64 ref numerics.  However, with
```

Comment:
if we're going to expand maybe we should actually have a doc somewhere instead of just a longer config str